packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagegroup/_mixins.scss

```diff
@@ -38,4 +42,15 @@
     font-size: $information-font-size;
     color: $information-author-name-color;
   }
+
+  .dx-chat-message-edited {
```

Comment:
When time is long its not elipsis 
<img width="603" alt="image" src="https://github.com/user-attachments/assets/5e47f9c5-6dd6-4d28-be83-e7af6a74a55b" />